src/components/Location/index.jsx

```diff
@@ -51,69 +28,49 @@ const Location = ({ barebone = false }) => {
   const biomassTaskResults = useSelector(get.biomassTaskResults);
   const polyDrawTooBig = useSelector(get.polyDrawTooBig);
   const biomassFetchIsFailed = useSelector(get.biomassFetchIsFailed);
-  const field = useSelector(get.field);
-  const { historyState, userHistoryList } = useSelector(get.user);
 
-  const [fieldName, setFieldName] = useState(field);
-
-  // api for getting biomass map
+  // API for getting biomass map
   useFetchHLS();
 
-  const isFieldNameExisted = () => {
-    if (historyState === historyStates.imported) return false;
-    const result = userHistoryList.find((history) => history.label === 'history-'.concat(fieldName));
-    return result !== undefined;
-  };
-
   return (
     <Box sx={{ width: '100%', padding: '0rem' }}>
       <Box mb={-2}>
-        <CustomizedAccordion defaultExpanded>
-          <AccordionSummary expandIcon={<ExpandMoreIcon />} aria-controls="panel1a-content" id="panel1a-header">
-            {!barebone && (
-              <Typography variant="h5" gutterBottom>
-                Where is your Field located?
-              </Typography>
-            )}
-          </AccordionSummary>
-          <AccordionDetails>
-            <Stack mb={1}>
-              <Typography variant="h8" gutterBottom>
-                Enter your address or zip code to determine your field&apos;s location. You can then zoom in and click to pinpoint it on the map. If
-                you know your exact coordinates, you can enter them in search bar separated by comma (ex. 37.7, -80.2 ).
-              </Typography>
-              {isSatelliteMode && (
-                <Typography variant="h8" gutterBottom pt={1}>
-                  Specify your field&apos;s boundary on the map using the drawing tool.
-                </Typography>
-              )}
-            </Stack>
-            <Box mb={2}>
-              <PSATextField
-                label="Name your Field (optional)"
-                value={fieldName}
-                onChange={(e) => setFieldName(e.target.value)}
-                error={isFieldNameExisted()}
-                helperText={isFieldNameExisted() ? 'Field name existed!' : null}
-                onBlur={() => {
-                  if (!isFieldNameExisted()) dispatch(set.field(fieldName));
-                }}
-                onKeyDown={(e) => {
-                  if (e.key === 'Enter' && !isFieldNameExisted()) dispatch(set.field(fieldName));
-                }}
-              />
-              <Help />
-            </Box>
-            <Stack mt={5} gap={1}>
-              {isSatelliteMode && (
+        <PSAAccordion
+          expanded
```

Comment:
Can you create a state for `expanded` to control it? Currently the accordion is always expanded